src/lib/api.js

```diff
@@ -53,12 +53,44 @@ export default async function API(): Promise<ApolloClient> {
 		}
 	} );
 
-	const httpLink = new HttpLink( { uri: API_URL, headers, fetchOptions: {
-		agent: createSocksProxyAgent(),
-	} } );
+	const proxyAgent = createSocksProxyAgent();
 
-	return new ApolloClient( {
+	const httpLink = new HttpLink( {
+		uri: API_URL,
+		headers,
+		fetchOptions: {
+			agent: proxyAgent,
+		},
+	} );
+
+	const apiClient = new ApolloClient( {
 		link: errorLink.concat( httpLink ),
 		cache: new InMemoryCache(),
 	} );
+
+	/**
+	 * Call the Public API with an arbitrary path (e.g. to connect to REST endpoints)
+	 * @param {*} path API path to pass to `fetch` -- will be prefixed by the API_HOST
+	 * @param {*} options options to pass to `fetch`
+	 * @returns {Promise} Return value of the `fetch` call
+	 */
+	apiClient.apiFetch = ( path: string, options = {} ) =>
```

Comment:
How does everyone feel about this helper to make an authenticated call to the public API outside the GraphQL endpoint? I didn't see a handy way to do it if it exists.

Reply:
This seems like a fine solution to me!